spec/addressable/template_spec.rb

```diff
@@ -209,7 +209,7 @@
       :path => "/foo/bar",
       :semi => ";",
       :list => %w(red green blue),
-      :keys => {"semi" => ';', "dot" => '.', "comma" => ','}
+      :keys => {"semi" => ';', "dot" => '.', :comma => ','}
```

Comment:
I noticed this specs are weird to write (you need to write a big array of expected outputs in different orders, probably legacy ruby 1.8 stuff when hash where unordered). So I just modified existing specs here to make it simple.

I'll probably make another PR to simplify those specs after I confirm it was only for ruby 1.8.